source/01-global/content-placeholder/content-placeholder.js

```diff
@@ -0,0 +1,18 @@
+// Used for to show simple demo content placeholders within style guide.
```

Comment:
This should have a `.jsx` extension since it exports a JSX/React component

Reply:
Making that changed fixed the ESLint error. Thanks!